.github/actions/windows-package/scripts/install_wix_cli.ps1

```
@@ -3,43 +3,170 @@ param()
 
 $ErrorActionPreference = 'Stop'
 
-$toolsDir = Join-Path $env:USERPROFILE '.dotnet\tools'
-if (-not ($env:PATH -split ';' | Where-Object { $_ -eq $toolsDir })) {
-    $env:PATH = "$toolsDir;$env:PATH"
-}
+function Set-ToolsPath {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Set-ToolsPath' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)